web/dispers/dispers.go

```diff
@@ -0,0 +1,48 @@
+package dispers
+
+import (
+	"net/http"
+
+	"github.com/cozy/echo"
+)
+
+/*
+*
+*
+CONCEPT INDEXOR'S ROUTES : those functions are used on route ./dispers/conceptindexor/
+*
+*
+*/
+
+func createConcept(c echo.Context) error {
+
+	// TODO: Get concept from body
+
+	// TODO: Decrypt concepts
+
+	// TODO: Create array of hashes
+
+	// TODO: Call HashMeThat for each concept
+	hash, err := enclave.HashMeThat(concept)
```

Comment:
`enclave` is not imported? 
This is nitpicking, but HashMeThat seems a bit familiar, I would prefer something like HashConcept :)

Reply:
enclave is not imported because not existent in this branch.

Yes, I agree with you. I just changed that in another branch. Sorry, I did not apply this change in this branch. I chose CreateConcept as a new name, is it ok for you ?

Reply:
Yep fine with me